nixos/tests/iscsi-root.nix

```diff
@@ -0,0 +1,166 @@
+import ./make-test-python.nix (
+  { pkgs, lib, ... }:
+    let
+      initiatorName = "iqn.2020-08.org.linux-iscsi.initiatorhost:example";
+      targetName = "iqn.2003-01.org.linux-iscsi.target.x8664:sn.acf8fd9c23af";
+    in
+      {
+        name = "iscsi";
+        meta = {
+          maintainers = pkgs.lib.teams.deshaw.members
+          ++ (with pkgs.lib.maintainers; [ ajs124 ]);
+        };
+
```

Comment:
Hopefully @ajs124 agrees to this :)

Reply:
Could you maybe also add yourself? Or are you not planning on using this, but just fixed it for $reasons.